### DISCUSSION TITLE
Specify Close Code + Reason on closing a client's socket

### DISCUSSION THREAD

**Discussion by xrnss** [uNetworking/uWebSockets.js#358] [Q&A] [answered]
I've looked around a bit and I can't seem to fine anything about this. Sorry if this is a duplicate, from what I can find, it isn't really mentioned anywhere. Hopefully some clearer docs come out if this feature does exist.

**Comment by hst-m** [comment, marked as answer]
the [docs](https://unetworking.github.io/uWebSockets.js/generated/interfaces/websocket.html#end) say to use `ws.end(code, reason)` I tested and is working

**Comment by xrnss** [comment]
Thanks for the help, I missed it. Sorry for being blind!
